docs/concepts.rst

```diff
@@ -228,6 +228,101 @@ Not like this, where the join task is skipped
 
 .. image:: img/branch_bad.png
 
+SubDAGs
+'''''''
+
+SubDAGs are perfect for repeating patterns. Defining a function that returns a
+DAG object is a nice design pattern when using Airflow.
+
+Airbnb uses the *stage-check-exchange* pattern when loading data. Data is staged
+in a temporary table, after which data quality checks are performed against
+that table. Once the checks all pass the partition is moved into the production
+table.
+
+As another example, consider the following DAG:
+
+.. image:: img/subdag_before.png
+
+We can combine all of the parallel ``task-*`` operators into a single SubDAG,
+so that the resulting DAG resembles the following:
+
+.. image:: img/subdag_after.png
+
+Note that SubDAG operators should contain a factory method that returns a DAG
+object. This will prevent the SubDAG from being treated like a separate DAG in
+the main UI. For example:
+
+.. code:: python
+
+  #dags/subdag.py
+  from airflow.models import DAG
+  from airflow.operators import DummyOperator
+
+
+  # Dag is returned by a factory method
+  def sub_dag(start_date, schedule_interval):
+    dag = DAG(
+      'main_dag.dummy_operator',
```

Comment:
It will be good to stick to the `parent_dag`, `child_dag` terminology that is used in other parts of this documentation. Also, as per my other comments, it would be good to pass the parent dag's name as an argument to the `sub_dag()` function instead of hard-coding the dag name to `main_dag.dummy_operator`. This will make the sub_dags generators more reusable across parent dags.